src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import lotto.global.ErrorMessage;
+
+public class Lotto {
+    private static final int LOTTO_SIZE_STANDARD = 6;
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validateNumbers(numbers);
+        this.numbers = List.copyOf(numbers);
```

Comment:
```java
    /**
     * Returns an <a href="#unmodifiable">unmodifiable List</a> containing the elements of
     * the given Collection, in its iteration order. The given Collection must not be null,
     * and it must not contain any null elements. If the given Collection is subsequently
     * modified, the returned List will not reflect such modifications.
     *
     * @implNote
     * If the given Collection is an <a href="#unmodifiable">unmodifiable List</a>,
     * calling copyOf will generally not create a copy.
     *
     * @param <E> the {@code List}'s element type
     * @param coll a {@code Collection} from which elements are drawn, must be non-null
     * @return a {@code List} containing the elements of the given {@code Collection}
     * @throws NullPointerException if coll is null, or if it contains any nulls
     * @since 10
     */
    static <E> List<E> copyOf(Collection<? extends E> coll) {
        return ImmutableCollections.listCopy(coll);
    }
```

위와 같이 불변 컬렉션으로 사용되어야 함을 명시하고 싶었습니다 😄